code/style.css

```diff
@@ -1,7 +1,210 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* { box-sizing: 
+  border-box; margin: 0
+ } 
+body {
+  
+  font-family: sans-serif; 
+}
+
+.logo{
+  display: none;
+}
+.navigation{
+  display: flex;
+  justify-content: space-evenly;
+  padding-top: 20px;
+  align-items: center;
+}
+.navigation-link{
+  color: rgb(197, 102, 39);
+ 
+}
+.hero {
+  /* Sizing */
+  width: 100%;
+  height: 100vh;
+
+
+/* Flexbox stuff */
+display: flex;
+   justify-content: center;
+   align-items: center;
+
+ /* Text styles */
+ text-align: center;
+}
+
+.hero::after{
+  width: 100%;
+  height: 100%;
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: rgba(23, 24, 18, 0.3);
+}
+
+  /* Background styles */
+  video {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    position: Absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: -1;
+    background-attachment:fixed;
+    margin: 0;
+}
+
+  .hero h1 {
+        /* Text styles */
+        font-size: 5em;
+        margin-top: 0;
+        margin-bottom: 0.5em;
+    }
+
+   
+      
+     
+   
+    .signupForm{
+      display: flex;
+      justify-content: center;
+      text-align: center;
+      margin: 5%;
+    }
+  
+  
+
+    form{
+      background-color: #ede7e4;
+      /* box-sizing: border-box; */
+      padding: 20px;
+      color: black;
+      margin-bottom: 30px;
+      margin-right: 20px;
+      border-radius: 30px;
+      
+    }
+
+  .main-content{
+    background-color: aliceblue;
+}
+
+input {
+  font-size: 20px;
+}
+
+input:valid{
+  background-color: rgb(53, 126, 59);
+}
 
-.active + .answer {
-  /* some styling here */
+input:invalid{
+  background-color: rgb(246, 97, 97); 
+}
+
+label{
+  display: block;
+  margin-bottom: 20px;
+}
+
+
+
+
+/* styling of the tablet and desktop!*/
+@media (min-width:768px) {
+ 
+  .logo{
+    display: flex;
+  }
+}
+
+@media (min-width: 1040px){
+.navigation .navigation-link {
+    text-align: center;
+    display: inline;
+}
+
+
+/*this section is for  my FAQ accordion. :D */
+  
```

Comment:
Why not place this above the media queries, since it's general style. It was a bit confusing to me when it came after the new styles for the different view ports.

Reply:
Changed this! I forgot to put it above the @media